doc/whats_new.rst

```diff
@@ -70,6 +70,8 @@ BUG
 
     - Fix a bug in :meth:`mne.time_frequency.AverageTFR.plot` when plotting without a colorbar by `Jaakko Leppakangas`_
 
+    - Fix ``_filenames`` attribute in creation of :class:`mne.io.RawArray` with :func:`mne.preporcessing.ica._sources_as_raw` by `Paul Pasler`_
+
```

Comment:
Btw., back here, I think the wording for this message is not optimal. What you actually fix is the 'filenames' field in the measurement info.